apps/consumers.py

```diff
@@ -19,16 +19,20 @@
 from rest_framework.renderers import JSONRenderer
 
 from apps.authentication import AsyncJsonAuthConsumer
-from .models import AsyncJob
-from .serializers import AsyncJobSerializer
+
+from apps.jobs.models import AsyncJob
+from apps.shipments.models import TrackingData
+
+from apps.jobs.serializers import AsyncJobSerializer
 
 
 class EventTypes(Enum):
     error = 0
     asyncjob_update = 1
+    new_tracking_data = 2
```

Comment:
Let's keep this consistent
```suggestion
    trackingdata_update = 2
```